### DISCUSSION TITLE
Des DMS have plan to support external identity provider via OAuth2 /OIDC/ SAML

### DISCUSSION THREAD

**Discussion by WestFarmer** [docker-mailserver/docker-mailserver#3580] [Q&A]
from [roundcube wiki](https://github.com/roundcube/roundcubemail/wiki/Configuration%3A-OAuth2)
>  XOAUTH support for both IMAP and SMTP servers Roundcube connects to

does DMS's underlying IMAP and SMTP server support XOAUTH ?

**Comment by polarathene** [comment]
There is an existing issue and WIP PR open regarding OIDC.

AFAIK, no one has enough time to work on the feature support presently so it's slow progress. Maybe next year,
